free-courses-en.md

```diff
@@ -281,7 +281,7 @@
 * [Princeton Algorithms, Part 1](https://www.coursera.org/course/algs4partI)
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046) (Udacity)
 * [Spring 5 Core - An Ultimate Guide](https://www.udemy.com/learn-spring-5-core-from-scratch/) - Somnath Musib (Udemy)
-* [Java 3 Month Program](https://www.youtube.com/watch?v=4hwmRUup7BE&list=PL7LtGuh_nbfphuB58q2SQ7C9k52THkRoN) - Bright Boost (Youtube)
+* [Java 3 Month Program](https://www.youtube.com/playlist?list=PL7LtGuh_nbfphuB58q2SQ7C9k52THkRoN) - Bright Boost (YouTube)
```

Comment:
In reference to [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file, this needs to be alphabetical. 

Simply moving your resource" Java 3 Month Program" to be right after the "Introduction to Java" resource would make the Travis CI build pass!